packages/masonry/docs/base.md

```diff
@@ -11,31 +11,54 @@ Make sure that all elements have the final sizes upon rendering. Image sizes sho
 
 This feature **does not** include a grid system - so you must use your own.
 
+You can add a masonry-identifier as a data-attribute to trigger events via the eventhub using `'${masonryIdentifier}:${method}'`
+
+Currently only the `appended` method is supported: `'${masonryIdentifier}:appended'`
+> Please note: Gaps can occure when items are lazily added. Consider this when designing the different item sizes.
+
 > In case you want to add filters you should check out [Isotope](https://isotope.metafizzy.co/)
 
 ```types.js
 require(['featurify'], function(featurify) {
   featurify([{ name: 'feature-masonry', path: 'packages/masonry/lib/main.min.js' }, 'base', 'base.features'], function(Masonry, base, features) {
     features.add('masonry', Masonry.default);
     features.init(document.body);
+
+    document.getElementById('append').addEventListener('click', () => {
```

Comment:
Don't use ES6 syntax - the code in the docs is not being transpiled.
This fragment will therefore not work in IE11.